docker/Dockerfile.binary

```diff
@@ -9,9 +9,11 @@ RUN cargo build --target x86_64-unknown-linux-musl --release
 FROM alpine:latest
 
 # Handle signal handlers properly
-RUN apk add --no-cache tini
+RUN apk add --no-cache tini postgresql-client
 COPY --from=cargo-build /usr/src/oba-services/target/x86_64-unknown-linux-musl/release/orderbook /usr/local/bin/orderbook
 COPY --from=cargo-build /usr/src/oba-services/target/x86_64-unknown-linux-musl/release/solver /usr/local/bin/solver
 
+COPY database usr/database
```

Comment:
This is probably no longer needed with the move to flyway in a separate container.